cms/djangoapps/contentstore/views/tests/test_entrance_exam.py

```diff
@@ -37,6 +39,56 @@ def setUp(self):
         milestones_helpers.seed_milestone_relationship_types()
         self.milestone_relationship_types = milestones_helpers.get_milestone_relationship_types()
 
+    def test_entrance_exam_milestone_addition(self):
+        """
+        Unit Test: test addition of entrance exam milestone content
+        """
+        parent_locator = unicode(self.course.location)
+        created_block = create_xblock(
+            parent_locator=parent_locator,
+            user=self.user,
+            category='chapter',
+            display_name=('Entrance Exam'),
+            is_entrance_exam=True
+        )
+        add_entrance_exam_milestone(self.course.id, created_block)
+        content_milestones = milestones_helpers.get_course_content_milestones(
```

Comment:
Can we check course has a content milestone with `REQUIRES` relationship type also?